packages/voice/src/app/agent/chat.tsx

```diff
@@ -675,8 +675,10 @@ export class WebRtcChatManager implements ChatManager {
     const audioTrack = track as RemoteAudioTrack;
     audioTrack.on(TrackEvent.AudioPlaybackStarted, () => console.log(`[chat] audio playback started`));
     audioTrack.on(TrackEvent.AudioPlaybackFailed, (err) => console.error(`[chat] audio playback failed`, err));
+    audioTrack.setAudioContext(this.audioContext);
     audioTrack.attach(this.audioElement);
     this.outAnalyzer = new StreamAnalyzer(this.audioContext, track.mediaStream!);
+    this.changeState(ChatManagerState.SPEAKING);
```

Comment:
Probably only want to do this in the case you mention below, right?

Reply:
Updated.